lib/utils/hasNamedColor.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const HAS_NAMED_COLOR =
```

Comment:
This one should be imported either from `/lib/reference/` or from a third party package.

Reply:
Does this still apply given the context that it needs to be exactly this regexp and that this regexp is only useful for this function?

Reply:
My rationale is that the W3C often add new names so it either needs to be shared in reference directory so that it can be reused by other rules or we want to push the responsibility of the maintenance of the list to an npm package. 

see #6712

Reply:
I get that, but the named colors is a pretty fixed set at this point.
`rebeccapurple` is the most recent addition and it has been in browsers since ±2014.

That is almost 10 years without a new named color :)

The system colors and color keywords are something else.

Reply:
I've moved this to the `/lib/reference/`.